v3/plugins/redhat/quarkus/1.2.0/meta.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+publisher: redhat
+name: quarkus
+version: 1.2.0
+type: VS Code extension
+displayName: Quarkus Tools
+title: Quarkus Tools by Red Hat
+description: This plug-in provides support for Quarkus development via a Quarkus language server, Java jdt.ls and Java Debugger.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/redhat-developer/vscode-quarkus
+category: Language
+firstPublicationDate: "2019-11-25"
+spec:
+  containers:
+    - image: "docker.io/eclipse/che-remote-plugin-runner-java8:next"
+      name: vscode-quarkus
+      memoryLimit: "1500Mi"
```

Comment:
Per discussion in https://github.com/eclipse/che-devfile-registry/pull/144#event-2900270521 is 1.5G enough memory?

Reply:
i didn't try big projects, but for https://github.com/quarkusio/quarkus-quickstarts.git samples 1.5G is enough